solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -274,6 +282,15 @@ public SolrResourceLoader getResourceLoader() {
     return resourceLoader;
   }
 
+  public SolrResourceLoader getResourceLoader(String pkg) {
```

Comment:
String is generic and so the method signature is non-obvious.  Rename to getResourceLoaderForPackage  ?